src/main/scala/sbtorgpolicies/settings/bash.scala

```diff
@@ -63,7 +63,7 @@ trait bash {
 
         val ref = extracted.get(thisProjectRef)
 
-        extracted.runAggregated[Unit](publishSigned in Global in ref, st)
+        extracted.runAggregated[Unit](publish in Global in ref, st)
```

Comment:
I'm not sure, but this would only affect to the `orgPublishRelease` task

Reply:
In sbt 1.0:

All downstream projects that use `orgAfterCISuccess` depend on `orgPublishReleaseTask`/`orgPublishRelease`, as far as I can tell. If `orgAfterCISuccess` is called on "master" with a `SNAPSHOT` version, `publishSigned` will break for the second push/merge because `overwrite = false`.

https://github.com/sbt/sbt-pgp/blob/master/pgp-plugin/src/main/scala-sbt-1.0/Compat.scala#L72